### PR TITLE
wdl4s 0.15 compatibility

### PR DESCRIPTION
**Pull request by davidangb**
DO NOT MERGE to develop - this PR is just an illustration of the changes I made.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -7,9 +7,9 @@ organization := "org.broadinstitute"
 
 scalaVersion := "2.11.11"
 
-val sprayV = "1.3.2"
+val sprayV = "1.3.4"
```
Comment (davidangb) [inline build.sbt:10]: based on the instructions at http://spray.io/project-info/current-versions/

File: build.sbt [hunk 2]
```diff
@@ -24,7 +24,7 @@ libraryDependencies ++= Seq(
   "org.apache.commons" % "commons-compress" % "1.4.1",  // upgrading a transitive dependency to avoid security warnings
   "org.apache.httpcomponents" % "httpclient" % "4.5.3",  // upgrading a transitive dependency to avoid security warnings
   "ch.qos.logback" % "logback-classic" % "1.2.2",
-  "com.github.swagger-spray" %% "swagger-spray" % "0.8.0",
+  "com.github.swagger-spray" %% "swagger-spray" % "0.8.0" excludeAll ExclusionRule(organization = "io.spray"),
```
Comment (davidangb) [inline build.sbt:27]: swagger-spray has a dependency on spray-routing, which caused a conflict with spray-routing-shapeless23.

File: build.sbt [hunk 3]
```diff
@@ -35,12 +35,12 @@ libraryDependencies ++= Seq(
...
   "com.zaxxer" % "HikariCP" % "2.3.9",
+  "org.broadinstitute" %% "wdl4s" % wdl4sV,
   "io.spray" %% "spray-can" % sprayV,
   "io.spray" %% "spray-client" % sprayV,
-  "io.spray" %% "spray-json" % "1.3.1", // NB: Not at sprayV. 1.3.2 does not exist.
-  "io.spray" %% "spray-routing" % sprayV,
+  "io.spray" %% "spray-json" % "1.3.3", // NB: Not at sprayV. 1.3.4 does not exist.
+  "io.spray" %% "spray-routing-shapeless23" % sprayV,
   "mysql" % "mysql-connector-java" % "5.1.42",
-  "org.broadinstitute" %% "wdl4s" % wdl4sV,
```
Comment (davidangb) [inline build.sbt:43]: whoops I moved this line and it probably didn't need to move.

File: project/plugins.sbt [hunk 4]
```diff
@@ -13,3 +13,5 @@ addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.0.4")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0")
 
 addSbtPlugin("io.spray" % "sbt-revolver" % "0.8.0")
+
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
```
Comment (davidangb) [inline project/plugins.sbt:17]: whoops I left this in my commit and it doesn't really need to be there.

**Comment by aednichols** [comment]
Superseded by https://github.com/broadinstitute/agora/pull/222

**Pull request closed without merging**